### PR TITLE
[FRONT-274] Fix Cypress tests

### PR DESCRIPTION
**Pull request by tumppi**
Fix missing `regeneratorRuntime` and permissions to be handled with Ethereum identities.

PS. Do you have problems getting Cypress tests to run consistently? I seem to get a couple of failing test cases per run with timeouts and It's not the same cases every time. They will pass when retrying the run.

### CODE REVIEW

File: app/cypress/support/commands.js [hunk 1]
```diff
@@ -8,22 +8,33 @@ import getClientConfig from '$shared/utils/getClientConfig'
...
+    const wallet = getWallet(mnemonic)
+    return wallet.getAddressString()
+}
+
 Cypress.Commands.add('login', (mnemonic = 'tester one') => (
     new StreamrClient({
         restUrl: 'http://localhost/api/v1',
         auth: {
-            privateKey: generatePrivateKey(`auto generated user ${mnemonic}`),
+            privateKey: generatePrivateKey(mnemonic),
```
Comment (juhah) [inline app/cypress/support/commands.js:37]: I guess for testing it's enough to only two words for entropy?

File: app/cypress/support/commands.js [hunk 2]
```diff
@@ -8,22 +8,33 @@ import getClientConfig from '$shared/utils/getClientConfig'
...
+        const wallet = getWallet(mnemonic)
         cache[mnemonic] = wallet.getPrivateKey().toString('hex')
     }
-
     return cache[mnemonic]
 }
 
+const getAddress = (mnemonic = 'tester one') => {
+    if (!cache[mnemonic]) {
+        generatePrivateKey(mnemonic)
```
Comment (juhah) [inline app/cypress/support/commands.js:26]: This `if` doesn't seem like it's needed?

File: app/cypress/integration/StreamPage.spec.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ import qs from 'query-string'
 import { matchPath } from 'react-router-dom'
 import uuid from 'uuid'
 
+// This is needed to get around missing regeneratorRuntime
+require('@babel/polyfill')
```
Comment (juhah) [inline app/cypress/integration/StreamPage.spec.js:6]: 👍

**Review by juhah — APPROVED** [review]

File: app/cypress/support/commands.js (hunk 1 above)
Comment (tumppi) [reply app/cypress/support/commands.js:37]: I think so. Other reason is that we can reference to other users easier with bare mnemonics. Of course we could wrap mnemonic handling with a function but I think this is easier and simpler.

Comment (juhah) [reply app/cypress/support/commands.js:37]: Yea, it's good like this 👍

File: app/cypress/support/commands.js (hunk 2 above)
Comment (tumppi) [reply app/cypress/support/commands.js:26]: True, it's prematurely cached. I'll remove it.

**Pull request merged**